Shaders/PBR/private/ComputeIrradianceMap.psh

```diff
@@ -38,14 +56,16 @@ float3 IrradianceMap(float3 N)
         float OmegaS = 1.0 / (float(g_NumSamples) * pdf);
         
         // Solid angle of 1 pixel across all cube faces
-        float OmegaP = 4.0 * PI / (6.0 * g_EnvMapDim * g_EnvMapDim);
+        float OmegaP = 4.0 * PI / (6.0 * g_EnvMapWidth * g_EnvMapHeight);
```

Comment:
For sphere map, 6.0 should be removed

Reply:
Are you sure we only need to remove this constant? I haven't looked into how the formulas for calculating mip levels are derived

Reply:
`6.0 * g_EnvMapDim * g_EnvMapDim` is the total number of pixels in the cubemap. Clearly, for the sphere map the number of pixels is `g_EnvMapDim * g_EnvMapDim`